src/pages/Checklist/index.tsx

```diff
@@ -234,16 +239,42 @@ export const Checklist = () => {
           <CheckBox
             label={
               <Text className="checkbox-label" style={{ display: 'inherit' }}>
-                For Ubuntu 20.04:
+                For Ubuntu 20.04, run
                 <pre className="my0">
+                  {' '}
                   <span style={{ color: colors.red.medium }}>
-                    {' '}
                     sudo timedatectl set-ntp on
                   </span>
                 </pre>
+                to check the system time.
+              </Text>
+            }
+          />
+          <ul className="sub-checklist-item">
+            <li className="py5">
+              <Text>
+                If you see error message{' '}
+                <code>Failed to set ntp: NTP not supported</code>, you may need
+                to install <code>chrony</code> or <code>ntp</code> package.
+              </Text>
+            </li>
+          </ul>
+          <CheckBox
+            label={
+              <Text className="checkbox-label" style={{ display: 'inherit' }}>
+                I've verified my server's time matches the wall clock.
               </Text>
             }
           />
+          <ul className="sub-checklist-item">
+            <li className="py5">
+              <Text>
+                <strong>Note:</strong> time can be set to UTC instead of
+                timezone especially in a VM which has clock configured for
```

Comment:
What I meant here is that Windows, amongst a bunch of other crazy decisions w.r.t. timezones & timing, sets the RTC to local time and the calculates UTC from there when it is needed. This means that VMs often have an option to have the RTC be in UTC or local time and is some thing to heed. (This is why the `RTC in local TZ` field exists in the `timedatectl` params described above.

Reply:
Thanks for the clarification!

I updated this note to

"Note: the RTC (Real-Time Clock) time may be set to UTC time instead of your local timezone especially in a VM which has clock configured on Windows."

Does that make sense to you?

Reply:
Yes, but it is the other way round, on windows, the RTC is in local, and on all sane OSs, it is in UTC.